### DISCUSSION TITLE
Bril syntax-highlighting extension

### DISCUSSION THREAD

**Discussion by yanggeorge** [sampsyo/bril#334] [General]
I have published a [Bril syntax-highlighting extension](https://marketplace.visualstudio.com/items?itemName=yanggeorge.bril-syntax-hightlighting).
Please have a try.

**Comment by Pat-Lafon** [comment]
Very cool!

I'm curious about the definition of `comments` in the language-configuration file versus `comment` in the tmLanguage file. Do you basically need to say that comments can exist both inside and outside of the language's begin/end points?

**Reply by yanggeorge** [reply]
VS Code offers two commands for comment toggling. Toggle Line Comment and Toggle Block Comment.

The `comments` in language-configuration.json is specified to control how VS Code should comment out lines. As for Bril, inserting  '#' to the beginning of each lines. 

The reference about language-configuration.json is [here](https://code.visualstudio.com/api/language-extensions/language-configuration-guide#comment-toggling)

**Comment by sampsyo** [comment]
Looks awesome! Thanks for sharing a link!
